README.md

```diff
@@ -70,6 +70,9 @@ delta_sharing.load_as_pandas(table_url)
 # If the code is running with PySpark, you can use `load_as_spark` to load the table as a Spark DataFrame.
 delta_sharing.load_as_spark(table_url)
 ```
+
+You can try this by running our [examples](examples/README.me) with the open, example Delta Sharing Server.
```

Comment:
nit: `examples/README.md` instead of `examples/README.me`?

Reply:
Yep! Thank you! Lol